### ISSUE TITLE
[ublock-v1.33.0] Error on empty tab after build

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#1767]
Recreate: build, then click ADN menu button on empty tab

> 9:48:17.728 TypeError: tab is undefined,  LinkHandlerParent.jsm:46:15
> 19:48:17.841 TypeError: json is null menu.js:86:5
> 19:48:17.948 TypeError: tab is undefinedLinkHandlerParent.jsm:46:15

at least the 2nd needs to be fixed (add an undefined check before referencing 'json' in menu.js)

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/737638/111304404-a9603800-8690-11eb-805c-3d23f6e2d9bc.png">

**Comment by mneunomne** [comment]
@dhowe could you please add more info? Could not reproduce.

**Comment by dhowe** [comment]
do you not see the error in the browser console (menu.js, line 86) when you build and run a new version of the addon and then click the menu on the initial empty tab ?

**Comment by mneunomne** [comment]
After a couple of tries, I was able to reproduce, it is a bit intermittent. But I just tested on master and there also seem to be the same issue.

The `details` of this messaging API call seems to be returning null, causing the issue in the `renderPage` function. 
https://github.com/dhowe/AdNauseam/blob/095825de278c4607df399b9f38862993bd7036e0/src/js/adn/menu.js#L434-L440

But perhaps this issue only happens in debug mode? wasn't able to reproduce outside of it.
